doc/python-devbox-setup.md

```diff
@@ -92,19 +109,25 @@ In order to setup your development environment to build the C binaries, you need
 ### Compile the Python modules
 The Python iothub_client and iothub_service_client modules support python versions 2.7.x, 3.4.x, 3.5.x or 3.6.x. Know the appropriate version you would like to build the library with for the following instructions.
 
+1. Clone the Azure IoT Python SDK Repo
+  
+    ```
+    git clone --recursive https://github.com/Azure/azure-iot-sdk-python.git 
+    ```
+
 1. Ensure that the desired Python version (2.7.x, 3.4.x, 3.5.x or 3.6.x) is installed and active. Run `python --version` or `python3 --version` at the command line to check the version.
-2. Open a shell and navigate to the folder **build_all/mac** in your local copy of the repository.
-3. Run the `./setup.sh` script to install the prerequisite packages and the dependent libraries.
+1. Open a shell and navigate to the folder **build_all/mac** in your local copy of the repository.
+1. Run the `./setup.sh` script to install the prerequisite packages and the dependent libraries.
     * Setup will default to python 2.7
     * To setup dependencies for python version greater than 3, run `./setup.sh --python-version X.Y` where "X.Y" is the python version (e.g. 3.4, 3.5 or 3.6)
-4. Run the `./build.sh` script.
+1. Run the `./build.sh` script.
     * Build will default to python 2.7
     * To build with python version greater than 3, run `./build.sh --build-python X.X` where "X.Y" is the python version (e.g. 3.4, 3.5 or 3.6) 
-5. After a successful build, the `iothub_client.so` Python extension module is copied to the [**device/samples**][device-samples] and [**service/samples**][service-samples] folders. Visit these folders for instructions on how to run the samples.
+1. After a successful build, the `iothub_client.so` Python extension module is copied to the [**device/samples**][device-samples] and [**service/samples**][service-samples] folders. Visit these folders for instructions on how to run the samples.
```

Comment:
Please fix the numbering here

Reply:
I updated, but so you know, markdown renderer will convert 
1.
1.

to 
1.
2.